.travis.yml

```diff
@@ -49,13 +49,9 @@ addons:
       - doxygen-latex
       - dvipng
       - libusb-1.0-0-dev
-cache:
-  directories:
-  - $HOME/flann
-  - $HOME/vtk
-  - $HOME/qhull
-  - $HOME/doxygen
-before_install:
-  - bash .travis.sh install
+      - libqhull-dev
+      - libvtk5-dev
+      - libflann-dev
+      - doxygen
```

Comment:
This brings us an ancient version of Doxygen from 2011 (1.7.6 to be precise). Apart from gazillion bug fixes, one useful feature that is missing there is Markdown support. Back in the day I even made a PR (#526) to get a newer version. I'm not sure how much of PCL docstrings are actually written with Markdown, but the fact that I made that request suggests that there are at least some.